OsmAnd/src/net/osmand/plus/measurementtool/MeasurementToolFragment.java

```diff
@@ -123,16 +110,15 @@
 	private boolean gpxPointsAdded;
 	private ApplicationMode snapToRoadAppMode;
 
-	private boolean inMovePointMode;
-	private boolean inAddPointAfterMode;
-	private boolean inAddPointBeforeMode;
 	private boolean isInSnapToRoadMode;
 
 	private int selectedPointPos = -1;
 	private WptPt selectedCachedPoint;
 
 	private int positionToAddPoint = -1;
```

Comment:
Should be deleted (already in context)
private boolean isInSnapToRoadMode;
private int selectedPointPos = -1;
  	private WptPt selectedCachedPoint;
  
  	private int positionToAddPoint = -1;